src/p4p/server/raw.py

```diff
@@ -154,10 +178,23 @@ def open(self, value, nt=None, wrap=None, unwrap=None, **kws):
         self._wrap = wrap or (nt and nt.wrap) or self._wrap
         self._unwrap = unwrap or (nt and nt.unwrap) or self._unwrap
 
+        # Intercept all arguments that start with 'handler_open_' and remove them from
+        # the arguments that go to the wrap and send them instead to the handler.open()
+        post_kws = {x: kws.pop(x) for x in [y for y in kws if y.startswith("handler_open_")]}
```

Comment:
This, and the equivalent line 212 for `post()`, is probably one of the more important changes. It means that any arguments prefixed `handler_open_` will be passed to the handler (if it exists) and not to `_wrap()`. An earlier version of the code added a single new argument that enabled or disabled the handler altogether. But while a smaller change it was also much crude.